src/casp/services/_settings.py

```diff
@@ -87,6 +87,9 @@ class ProductionSettings(AllEnvSettings):
 class LocalSettings(AllEnvSettings):
     # General
     debug = True
+    MODEL_SERVER_URL: str = "http://localhost:8001"
+    API_SERVICE_PORT: int = 8000
+    MODEL_SERVICE_PORT: int = 8001
```

Comment:
This is maybe a silly question, but why is there are `MODEL_SERVICE_PORT` variable if the port is already set in `MODEL_SERVER_URL`?  Do they serve different purposes?

Reply:
Oh, this is because `MODEL_SERVER_URL` is used by api manager to access it, while `MODEL_SERVICE_PORT` is used by gunicorn to serve the model service app. I agree, it is a bit confusing tho.